blues/sparrow-reference-web-app
# feat: modifying with new lora signal icons

# Problem Context

The Sparrow's LoRa icons showing signal strength are hard to read at their current size, so we're replacing them with slightly chunkier, simpler icons that are more readable scaled down.

## Changes

Replace one set of LoRa icons with another set and modify their size and alignment

## Screenshot (if applicable)

![Screen Shot 2022-08-10 at 3 02 59 PM](https://user-images.githubusercontent.com/20400845/184005717-aaa01946-d750-44a7-8ff1-ec7ec9b964e1.png)

## Testing

Unit / integration / e2e tests?

No new automated tests added.

Steps to test manually?

1. Get Sparrow running locally and go to http://localhost:4000
2. In the `pages/[gatewayUID]/node/[nodeId]/details.tsx file, add the following code underneath the `type NodeDetailsData`
```
 const viewModel = { ...viewModel, barsIconPath: "2", barsTooltip: "2" };
```
3. Observe the new LoRa signal strength icons on the Node cards. Change the numbers from "0" to "4" to see the full range of icons and strengths.

Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

https://trello.com/c/nAWN13jm/97-update-sparrow-web-app-lora-icons
